docs/docs/css-modules.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: CSS Modules
+title: Locally-scoped styles with CSS Modules
```

Comment:
> It would be great to show an example on this page too, instead of shipping folks off to the tutorial.

Agreed! Do you think this should be done in this PR, or open an issue later to see if others want to contribute examples to these docs?

Reply:
> I think we need to set expectations a little more with CSS Modules on this page since they aren't consumed in quite the same way as traditional CSS. You could add something like this:
> ...

Added in f67a5bc

Reply:
woo! awesome!